pkg/threescale/backend/manager.go

```diff
@@ -212,6 +218,61 @@ func (b *Backend) Flush() {
 			}
 			service[i].authResp = resp
 		}
+
+		for _, cachedApp := range service {
+			app, ok := b.cache.Get(cachedApp.cacheKey)
+			if !ok {
+				continue
+			}
+			app.Lock()
+			if cachedApp.reportingErr {
+				app.Unlock()
+				continue
+			}
+
+			if cachedApp.authErr {
+				app.Unlock()
+				continue
+			}
```

Comment:
This is making no difference when reporting ok and auth failed (not sure what we agreed on at this point).

Reply:
yes, should have been some todo's there